docs/.vitepress/data/sidebar.ts

```diff
@@ -388,6 +388,10 @@ function sidebarDeveloperReference() {
 						},
 					],
 				},
+				{
+					link: '/extensions/sdk-composables',
+					text: 'App Extension Composables',
```

Comment:
Would that work too? 🤔 
```suggestion
					link: '/extensions/app-composables',
					text: 'App Composables',
```

It feels a bit long in the sidebar 😢 

<img width="259" src="https://github.com/directus/directus/assets/5363448/6e4310b0-b67f-41ce-9c6d-ecbdb20aa7ba">

Reply:
I also noticed the sidebar is getting stuffed...initially considered grouping some sections together, not entirely certain on which to group but I can revisit this.

Reply:
On a second thought, we could introduce a new section, for example:
```
Resources >
  Components (previously UI Components)
  Composables (previously App Extension Composables)
  Packages (previously Internal Packages)
```

Reply:
What do you think about this grouping:

Fundamentals or Getting Started>
   Introduction
   Installing Extensions
   Creating Extensions
Development >
  Developing Extensions
  Secure Extensions >
  Extension Services >
  App Composables
  UI Components
Reference >
   Packages

Reply:
I like Pascal's grouping of resources more, along with shortening names of pages within it. Then, I like Esther's overall structure. I think this works: 

Fundamentals
- Introduction
- Installing Extensions
- Creating Extensions

Developing Extensions
- Extension Types (prev 'Developing Extensions' dropdown of all the types)
- Secure Extensions
- Extension Services

Resources
- Components
- Composables
- Packages